src/functions/getResult/framework/handler.ts

```diff
@@ -13,7 +13,7 @@ import { gzipSync } from 'zlib';
 export async function handler(event: APIGatewayEvent, fnCtx: Context): Promise<Response> {
   await bootstrapConfig();
   try {
-    const appRefPathParam = event.pathParameters['app-ref'];
+    const appRefPathParam = parseInt(event.pathParameters['app-ref'], 10);
```

Comment:
Doesn't the linter complain about using an unqualified global function? I've always had to write `Number.parseInt` to make it happy.

Reply:
Nope it hasn't complained, have updated to `Number.parseInt()` to make it clearer. Maybe due to having different rules in the `package.json` script?